README.md

```diff
@@ -11,12 +11,12 @@ THIS IS RESEARCH CODE PROVIDED TO YOU "AS IS" WITH NO WARRANTIES OF CORRECTNESS.
 ## Contents
 1. [Software Dependencies and Installation](#software-dependencies-and-installation)
 2. [Jupyter Notebooks](#jupyter-notebooks)
-   - [Downloading GUNW products using ariaDownload.py - ariaDownload program tutorial](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaDownload/ariaDownload_tutorial.ipynb)
-   - [Manipulating GUNW products using GDAL (python and commandline) - GDAL basics tutorial](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/GDAL_basics/GDAL_basics.ipynb)
-   - [Manipulating Layers of ARIA standard GUNW products - ariaExtract.py program tutorial](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaExtract/ariaExtract_tutorial.ipynb)
-   - [Assessing data quality and spatiotemporal coverage of interferograms - ariaPlot.py program tutorial](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaPlot/ariaPlot_tutorial.ipynb)
-   - [Preparing ARIA standard GUNW products layers for time-series analysis using ariaTSsetup.py - ariaTSsetup.py program tutorial](https://nbviewer.jupyter.org/github/aria-tools/ARIA-tools-docs/blob/master/JupyterDocs/ariaTSsetup/ariaTSsetup_tutorial.ipynb)
-    - [External - Time-series InSAR processing using the Miami INsar Time-series software in PYthon tutorial](https://github.com/insarlab/MintPy)    
+   - [Downloading GUNW products using ariaDownload.py - ariaDownload program tutorial](#downloading-gunw-products-using-ariadownload.py---ariadownload-program-tutorial)
+   - [Manipulating GUNW products using GDAL (python and commandline) - GDAL basics tutorial](#manipulating-gunw-products-using-gdal-(python-and-commandline)---gdal-basics-tutorial)
+   - [Manipulating Layers of ARIA standard GUNW products - ariaExtract.py program tutorial](#manipulating-layers-of-aria-standard-gunw-products---ariaextract.py-program-tutorial)
+   - [Assessing data quality and spatiotemporal coverage of interferograms - ariaPlot.py program tutorial](#assessing-data-quality-and-spatiotemporal-coverage-of-interferograms---ariaplot.py-program-tutorial])
+   - [Preparing ARIA standard GUNW products layers for time-series analysis using ariaTSsetup.py - ariaTSsetup.py program tutorial](#preparing-aria-standard-gunw-products-layers-for-time-series-analysis-using-ariatssetup.py---ariatssetup.py-program-tutorial)
+    - [External - Time-series InSAR processing using the Miami INsar Time-series software in PYthon tutorial](#external---time-series-insar-processing-using-the-miami-insar-time-series-software-in-python-tutorial)
```

Comment:
@rzinke we had broken links here in the past. @bbuzz31 fixed them a few days ago and poitns to the viewer of the notebooks. At the moment his changes are being overwritten. Could you fix that?

Could you also add your notebooks to the table of content?